PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA.cxx

```diff
@@ -397,8 +397,7 @@ void AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA::DoJetLoop()
 			  AliEMCALTriggerPatchInfo *currentpatch(nullptr);
 			  for(TIter patchiter = TIter(triggerpatches).Begin(); patchiter != TIter::End(); ++patchiter){
 				  currentpatch = static_cast<AliEMCALTriggerPatchInfo *>(*patchiter);
-				  if(currentpatch->GetPatchSize() != 2) continue;  // Reject L0 patches
-
+				  if(currentpatch->GetPatchSize() != 2) continue;  // Reject L0 patche
```

Comment:
These are not necessarily only L0 patches - EGA patches also have patch size 2. Trigger patches encode the algorithm under which they are selected. I.e. jet patches can be selected via patch->IsJetHigh().